cross/arm-none-eabi-gcc/Portfile

```diff
@@ -4,10 +4,11 @@ PortSystem          1.0
 PortGroup           crossgcc 1.0
 PortGroup           compiler_blacklist_versions 1.0
 
-crossgcc.setup      arm-none-eabi 7.3.0
+crossgcc.setup      arm-none-eabi 7.4.0
 crossgcc.setup_libc newlib 3.0.0
 
-revision            4
+
```

Comment:
(Dirty nitpicking: do we need the extra space?)

Reply:
Sorry about that.
It should be fixed now.